Mathlib/NumberTheory/Cyclotomic/Three.lean

```diff
@@ -109,3 +113,79 @@ theorem eq_one_or_neg_one_of_unit_of_congruent (hcong : ∃ n : ℤ, λ ^ 2 ∣
     have : (hζ.pow_of_coprime 2 (by decide)).toInteger = hζ.toInteger ^ 2 := by ext; simp
     simp only [this, PNat.val_ofNat, Nat.cast_ofNat, mul_neg, Int.cast_neg, ← neg_add, ←
       sub_eq_iff_eq_add.1 hx, Units.val_neg, val_pow_eq_pow_val, IsUnit.unit_spec, neg_neg]
+
+variable (x : 𝓞 K)
+
+/-- Let `(x : 𝓞 K)`. Then we have that `λ` divides one amongst `x`, `x - 1` and `x + 1`. -/
+lemma dvd_or_dvd_sub_one_or_dvd_add_one : λ ∣ x ∨ λ ∣ x - 1 ∨ λ ∣ x + 1 := by
```

Comment:
Why not adding a `lambda` in front of the first `dvd`? Also, shouldn't this be also `private`?

Reply:
I fixed the name. It cannot be `private` since it will be needed in the proof of flt3, in another file (I will make sure to make "public" only the results I really need in the other file, maybe only this one).

Reply:
But if you make it private and then prepend the namespace, wouldn't this work?

Reply:
Are you thinking about `protected`? AFAIK the purpose of `private` is to make lemmas "invisible" in other files (or sections, or whatever).

Reply:
Oh right, I had `protected` in mind, and I have the same mental image of `private` as you have, so I finally agree that this should not be `private` (but hopefully in a namespace - I am not with VSCode now and cannot check).

Reply:
Yes, everything is in a (very specific) namespace.